File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/Python/s380356166_comments_removed.py
K, S = map(int, input().split())
ans = 0
for x in range(K + 1):
    for y in range(K + 1):
        wk = S - x - y
        if 0 <= wk <= K:
            ans += 1
print(ans)
# End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/Java/C++/s380356166_comments_removed.cpp
#include <iostream>
using namespace std;

int main() {
    int K, S;
    cin >> K >> S;
    int ans = 0;
    for (int x = 0; x <= K; ++x) {
        for (int y = 0; y <= K; ++y) {
            int wk = S - x - y;
            if (wk >= 0 && wk <= K) {
                ans++;
            }
        }
    }
    cout << ans << endl;
    return 0;
}
// End of Code
